gql_code_builder/lib/src/operation/data.dart

```diff
@@ -115,16 +139,17 @@ List<Class> _buildSelectionSetDataClasses({
         ..implements = ListBuilder(
             superclassSelections.keys.map<Reference>((superName) => refer(
                   superName,
-                  superName == parentName ? null : fragmentsDocUrl,
+                  superclassSelections[superName]
+                      .url
+                      ?.replaceAll(RegExp(r".graphql$"), ".data.gql.dart"),
```

Comment:
I temporarily hard-coded the string replacement here. Please let me know how you think we should pass in the extension.

Reply:
I'm trying to think of a way to solve this issue by using the custom import Allocator.

Please add a code comment here about it being hard-coded. I'll try to fix this when improving the allocator.